network-api/networkapi/donate/tests/test_landing_page.py

```diff
@@ -36,8 +36,8 @@ def test_subpage_types(self):
         Testing the DonateLandingPage's allowed subpage types.
         """
         self.assertAllowedSubpageTypes(
-            parent_model=pagemodels.DonateLandingPage,
-            child_models={pagemodels.DonateHelpPage},
+            parent_model=DonateLandingPage,
+            child_models={DonateHelpPage, OpportunityPage},
```

Comment:
Updating the test to account for new child page type. Also, updated file to just name models instead of using `pagemodels.<name_of_donate_page_model>`. Since we are now introducing pages that are not specifically donate related, I thought this would help make the file more readable.